### DISCUSSION TITLE
Importing 3D models into webots 2023a and splitting parts into shapes for a robot

### DISCUSSION THREAD

**Discussion by kubz2001** [cyberbotics/webots#5713] [General]
Hi, I am quite new with webots, I am wondering if there is a possibility to import 3D models into webots 2023a as it was possible within webots 2021b, I have managed to import it within 2021b but when I specify the bounding object and I apply physics the robot falls through the arena and bounces around underneath, same thing occurs when I just create a solid with a box shape and apply bounding object and physics to it.

**Comment by omichel** [comment]
Please consider this [question and answer](https://robotics.stackexchange.com/questions/24227/how-to-import-a-3d-model-in-webots-r2022b). Up-vote it if you believe it is helpful, otherwise, please add a comment or a new question there.

**Reply by kubz2001** [reply]
Yes I noticed that after, sorry, I will post it within Q&A. Thanks
